src/components/app.js

```diff
@@ -83,6 +85,8 @@ class App extends Component {
                         <Route path="/event" component={(props) => <Event user={this.state.user} />} />
                         <Route path="/eventlist" component={(props) => <EventList user={this.state.user} />} />
                         <Route path="/createevent" component={(props) => <CreateEvent user={this.state.user} />} />
+                        <Route path="/joinevent" component={(props) => <JoinEvent user={this.state.user} />} />
+                        <Route path="/eventinfo/:id,:name,:time,:desc,:ind,:participating" component={(props) => <EventInfo user={this.state.user} />} />
```

Comment:
This works in the short term but is not good in practice.

Just have :id and then add a componentDidMount for eventInfo that fetches all the information

Reply:
Actually having things render on the page that are retrieved from the url is a a huge security risk

Reply:
okay yeah i'll try to find a workaround for this